addons/hr_attendance/static/src/components/card_layout/card_layout.js

```diff
@@ -35,7 +28,22 @@ export class CardLayout extends Component {
         });
     }
 
+    getCurrentDate() {
+        return DateTime.now();
+    }
+
     getCurrentTime() {
-        return DateTime.now().toLocaleString(DateTime.TIME_SIMPLE);
+        return this.getCurrentDate().toLocaleString(DateTime.TIME_SIMPLE);
+    }
+
+    getDateTime() {
+        return {
+            dayOfWeek: this.getCurrentDate().toFormat("cccc"),
+            date: this.getCurrentDate().toLocaleString({
+                ...DateTime.DATE_FULL,
+                weekday: undefined,
+            }),
+            time: this.getCurrentTime(),
+        };
     }
```

Comment:
Oops, sorry to bother further, it could actually be done without the other two methods, which  could then be removed:
```suggestion
    getDateTime() {
        const now = DateTime.now();
        return {
            dayOfWeek: now.toFormat("cccc"),
            date: now.toLocaleString({
                ...DateTime.DATE_FULL,
                weekday: undefined,
            }),
            time: now.toLocaleString(DateTime.TIME_SIMPLE),
        };
    }
```

Reply:
makes sense of course, thank you.